ccmlib/scylla_rest.py

```diff
@@ -0,0 +1,27 @@
+
+import json
+import requests
+
+class ScyllaREST():
+    def __init__(self, ip = '127.0.0.1', port = 10000):
+        self.ip = ip
+        self.port = port
+        self.url = 'http://{}:{}'.format(ip, port)
+
+    def post(self, item, params):
+        headers = {
```

Comment:
It looks like the headers in `get` and `post` methods are same, so should be defined in the constructor